### ISSUE TITLE
Executing final e = <String, String>{}; returns exception 'Null check operator used on a null value'

### ISSUE THREAD

**Issue by al-tush** [ethanblake4/dart_eval#180]
Executing code 
`final e = <String, String>{};`
or any other with empty map returns
```
The following _TypeError was thrown building Builder:
Null check operator used on a null value

The relevant error-causing widget was: 
  MaterialApp MaterialApp:file:///C:/Projects-test/eval1/lib/main.dart:15:12
When the exception was thrown, this was the stack: 
#0      compileSetOrMapLiteral (package:dart_eval/src/eval/compiler/collection/set_map.dart:53:21)
#1      parseLiteral (package:dart_eval/src/eval/compiler/expression/literal.dart:46:12)
#2      compileExpression (package:dart_eval/src/eval/compiler/expression/expression.dart:32:12)
#3      compileVariableDeclarationList (package:dart_eval/src/eval/compiler/statement/variable_declaration.dart:34:17)
#4      compileVariableDeclarationStatement (package:dart_eval/src/eval/compiler/statement/variable_declaration.dart:15:3)
#5      compileStatement (package:dart_eval/src/eval/compiler/statement/statement.dart:23:12)
#6      compileBlock (package:dart_eval/src/eval/compiler/statement/block.dart:17:20)
#7      compileFunctionDeclaration (package:dart_eval/src/eval/compiler/declaration/function.dart:84:14)
```

**dart_eval**: 0.7.4

**Comment by ethanblake4** [comment]
Fixed in v0.7.5

**Comment by al-tush** [comment]
Thank you! But looks like this issue has a continuation :(
